firmware/application/file_path.cpp

```diff
@@ -22,28 +23,85 @@
 #include "file_path.hpp"
 #include "file.hpp"
 
-const std::filesystem::path adsb_dir = u"ADSB";
-const std::filesystem::path ais_dir = u"AIS";
-const std::filesystem::path apps_dir = u"APPS";
-const std::filesystem::path aprs_dir = u"APRS";
-const std::filesystem::path audio_dir = u"AUDIO";
-const std::filesystem::path blerx_dir = u"BLERX";
-const std::filesystem::path bletx_dir = u"BLETX";
-const std::filesystem::path captures_dir = u"CAPTURES";
-const std::filesystem::path debug_dir = u"DEBUG";
-const std::filesystem::path firmware_dir = u"FIRMWARE";
-const std::filesystem::path freqman_dir = u"FREQMAN";
-const std::filesystem::path gps_dir = u"GPS";
-const std::filesystem::path logs_dir = u"LOGS";
-const std::filesystem::path looking_glass_dir = u"LOOKINGGLASS";
-const std::filesystem::path playlist_dir = u"PLAYLIST";
-const std::filesystem::path remotes_dir = u"REMOTES";
-const std::filesystem::path repeat_rec_path = u"CAPTURES";
-const std::filesystem::path samples_dir = u"SAMPLES";
-const std::filesystem::path screenshots_dir = u"SCREENSHOTS";
-const std::filesystem::path settings_dir = u"SETTINGS";
-const std::filesystem::path spectrum_dir = u"SPECTRUM";
-const std::filesystem::path splash_dir = u"SPLASH";
-const std::filesystem::path sstv_dir = u"SSTV";
-const std::filesystem::path wav_dir = u"WAV";
-const std::filesystem::path whipcalc_dir = u"WHIPCALC";
+const std::filesystem::path adsb_dir = u"SYS/ADSB";  // this dir no need to seperate profiles since all are resources
+
+const std::filesystem::path ais_dir = u"SYS/AIS";  // this dir no need to seperate profiles since all are resources
+
+const std::filesystem::path apps_dir = u"APPS";  // currently in root
+
+// APRS
+const std::filesystem::path aprs_dir = u"APRS";  // fake path
```

Comment:
For each of these, the _dir and _dir_user values seem pretty similar; do we really need both?

Reply:
It's a little bit sad about this part, because the return value of scan_root_files method are returning the path without first /. So it's still different than std path.
And all the apps/ not std functions handling these all in different way, which you actually didn't fixed that very well (which i fixed for you in #2070).

I maybe will make them all complete, based on following conditions:
1. making sure current logic works
2. After this PR merged. 

The reason for skipping this part is because this PR is really big and it's too much to keep working on it. 

On my old computer, I need to refresh about 5 times to load the file changelist

sorry.